src/pages/Setting/Setting.jsx

```diff
@@ -1,28 +1,100 @@
-            <Form>
-                <ImageWrapper>
-                <ProfileImg src={avataaar}></ProfileImg>
-                <EditBtn htmlFor="picUpdate"><EditIcon /></EditBtn>
-                <input type="file" id="picUpdate" style={{display:"none"}} />
-                </ImageWrapper>
-                <Label htmlFor="username" name>Name</Label>
-                <InputField id="username" type="text"></InputField>
-                <Label htmlFor="email">Email</Label>
-                <InputField id="email" type="email"></InputField>
-                <Label htmlFor="bio">Bio</Label>
-                <BioInput id="bio" type="text" cols="28" rows="6"></BioInput>
-                <Label htmlFor="url">URL</Label>
-                <InputField id="url" type="text"></InputField>
-            <PrimaryButton>Update Profile</PrimaryButton>
-            </Form>
-        </Wrapper>
-    )
-}
\ No newline at end of file
+export const Setting = () => {
+  const { currentUser } = useSelector((store) => store.auth);
+  const {users} = useSelector((store) => store.users);
+
+  const user = users.find(user => user._id === currentUser._id)
+  
+  const [newData, setNewData] = useState({
+    firstName: user.firstName,
+    lastName:user.lastName,
+    link: user.link,
+    bio: user.bio,
+    userPhoto: user.userPhoto,
+    email: user.email,
+  });
+  const { firstName,lastName, link, bio, userPhoto, email } = newData;
+  const dispatch = useDispatch();
+  const changeHandler = (e) => {
+    setNewData((prev) => ({ ...prev, [e.target.name]: e.target.value }));
+  };
+  const clickHandler = () =>{
+      dispatch(editUser(newData)) 
+  }
+  return (
+    <Wrapper>
+      <Form>
+        <ImageWrapper>
+          <ProfileImg src={userPhoto}></ProfileImg>
+          <EditBtn htmlFor="picUpdate">
+            <EditIcon />
+          </EditBtn>
+          <input type="file" id="picUpdate" style={{ display: "none" }} />
+        </ImageWrapper>
+        <Label htmlFor="firstName">
+          First Name
+        </Label>
+        <InputField
+          id="firstName"
+          name="firstName"
+          type="text"
+          value={firstName}
+          onChange={(e) => changeHandler(e)}
+        ></InputField>
+        <Label htmlFor="lastName">
+          Last Name
+        </Label>
+        <InputField
+          id="lastName"
+          name="lastName"
+          type="text"
+          value={lastName}
+          onChange={(e) => changeHandler(e)}
+        ></InputField>
+        <Label htmlFor="email">Email</Label>
+        <InputField
+          id="email"
+          type="email"
+          value={email}
+          name="email"
+          onChange={(e) => changeHandler(e)}
+        ></InputField>
+        <Label htmlFor="bio">Bio</Label>
+        <BioInput
+          id="bio"
+          type="text"
+          name="bio"
+          cols="28"
+          rows="6"
+          value={bio}
+          onChange={(e) => changeHandler(e)}
+        ></BioInput>
```

Comment:
suggestion : this component has no fixed size because of this, it does not look good, you can fix its size using some CSS properties

Reply:
Ok Will do that in next PR or may be after completion of basic p0